spec/models/project/queue/classifiable_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'spec_helper'
+require 'models/project/queue_spec'
+
+RSpec.describe Project::Queue::Classifiable do
+  include_context 'Project::Queue context'
+
+  it_behaves_like 'Project::Queue'
+
+  describe '#next' do
```

Comment:
There'll probably be quite a lot of similarity when we implement `Project::Queue::Extractable`, but the results really depend on the specific requests that are in the queue. Open to ideas on extracting some of this into the shared examples.